repository/src/main/java/org/alfresco/repo/event2/filter/NodePropertyFilter.java

```diff
@@ -48,6 +48,9 @@ public class NodePropertyFilter extends AbstractNodeEventFilter
                                                                       ContentModel.PROP_CREATOR,
                                                                       ContentModel.PROP_CREATED,
                                                                       ContentModel.PROP_CONTENT);
+    // These properties should not be excluded from the properties object
+    private static final Set<QName> ALLOWED_PROPERTIES = Set.of(ContentModel.PROP_CASCADE_TX,
+                                                                ContentModel.PROP_CASCADE_CRC);
 
     private final List<String> nodeAspectsBlackList;
```

Comment:
Can you please rename this instance variable to `nodePropertiesBlackList`. I think it was simple copy and paste issue!

Reply:
Done.